src/Aliencube.AzureFunctions.Extensions.OpenApi.Core/DocumentHelper.cs

```diff
@@ -250,6 +250,30 @@ public OpenApiResponses GetOpenApiResponses(MethodInfo element, NamingStrategy n
             return schemes;
         }
 
+        /// <inheritdoc />
+        public IList<OpenApiSecurityRequirement> GetOpenApiSecurityRequirements(IDictionary<string, OpenApiSecurityScheme> securitySchemes)
```

Comment:
Renders as
![image](https://user-images.githubusercontent.com/55759342/89797907-861d4b00-daf9-11ea-92a9-67b5c6c6c5f6.png)

Reply:
@gsteendp Let me think about this. It's way bigger than this implementation. Can you split this PR into two parts - one for `OpenApiDocument` and the other for `OpenApiSecurityRequirement`?

Reply:
Reverted the security requirements aspect from this pull